components/NavBar.vue

```diff
@@ -12,8 +12,11 @@
 		<nuxt-link to="/our-data/">
 			Our Data
 		</nuxt-link>
-		<nuxt-link to="/news-and-views/">
-			News and Views
+		<nuxt-link to="/news/page/1">
```

Comment:
I believe you do not need the `/page/1`. Just `to="/news/"` should be fine.

Reply:
Redirecting to `/news/` no longer works because I've removed `/pages/news/index.vue`, which is pretty much a duplicate of `/pages/news/_num/index.vue` with the page number set to 1. Redirecting to `/news/page/1` has the same effect and saves us from maintaining an extra script. 

It's the same reason that "views" link is now pointing to `/views/page/1`.

Reply:
I'm mildly concerned about this because we end up with 404 (actually, a different error) for `/news/` and `/views/`. I think the convention with paginated content is that visiting the base path shows you the first page. @cindyli is there any way we can display the first page of content at the base URL and all subsequent paginated content at the `/page/<pagenumber>/` route? I understand this might require a duplicate layout but I think it's worth doing.